docs: update full static status (#6460)
nuxt/framework: docs/content/2.guide/5.deploy/2.static-hosting.md

# Static Hosting

::alert{type=info}

> The [full static feature](https://nuxtjs.org/announcements/going-full-static) is [not yet implemented](https://github.com/nuxt/framework/issues/6411) but will be in the near future.

Reviewer: ```suggestion
The [full static feature](https://nuxtjs.org/announcements/going-full-static) is [under development](https://github.com/nuxt/framework/issues/6411) and will be available in the near future.
```
